src/components/utils/src/qdb_wrapper/sql_query.cc

```diff
@@ -103,8 +103,6 @@ SQLQuery::SQLQuery(SQLDatabase* db)
 
 SQLQuery::~SQLQuery() {
   Finalize();
-  db_->Close();
-  delete db_;
```

Comment:
@BSolonenko Why did you delete it?

Reply:
@AByzhynar fixed in https://github.com/Ford-Luxoft/sdl_core/pull/11/commits/fade7796b2502a0c96694a7b372fb412d93f577c